### DISCUSSION TITLE
Accessing custom EV3 name in code.

### DISCUSSION THREAD

**Discussion by ChrisShepherd** [pybricks/support#260] [Q&A] [answered]
I'm building a project with several EV3s.  I've given each of them a unique custom name.

I want to download the same code to all of them, but I want each to pick and choose which code to use based on their name
If EV3 name is "One" then do this code
If EV3 name is "Two" then do this code
If EV3 name is "Three" then do this code

Is there a way to access this info on the brick?

**Comment by dlech** [comment, marked as answer]
Hi Chris,

Try using this function this to get the hostname:

```python
def hostname():
    with open('/etc/hostname') as f:
        return f.read().strip()
```

This assumes you used `sudo ev3dev-config` to give each EV3 a unique hostname.

**Reply by ChrisShepherd** [reply]
Worked perfectly!  Thank you very much!
